### PR TITLE
Dummy pr

### PR DESCRIPTION
**Pull request by seoky0**

### CODE REVIEW

**Comment by seoky0** [comment]
コメント

File: app/controllers/home_controller.rb [hunk 1]
```diff
@@ -0,0 +1,19 @@
+class HomeController < ApplicationController
+  def index; end
+
+  def new
```
Comment (seoky0) [inline app/controllers/home_controller.rb:4]: **[rubocop]** <sub>reported by [reviewdog](https://github.com/haya14busa/reviewdog) :dog:</sub>
Metrics/MethodLength: Method has too many lines. [13/10]

File: app/helpers/application_helper.rb [hunk 2]
```diff
@@ -1,2 +1,17 @@
 module ApplicationHelper
+  def hoge_helper
```
Comment (seoky0) [inline app/helpers/application_helper.rb:2]: **[rubocop]** <sub>reported by [reviewdog](https://github.com/haya14busa/reviewdog) :dog:</sub>
Metrics/MethodLength: Method has too many lines. [13/10]

File: app/controllers/home_controller.rb [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+    puts 'hoge'
+    puts 'hoge'
+    puts 'hoge'
+    puts 'hoge'
+    puts 'hoge'
+    puts 'hoge'
+    puts 'hoge'
+  end
+
+  def create
```
Comment (seoky0) [inline app/controllers/home_controller.rb:20]: **[rubocop]** <sub>reported by [reviewdog](https://github.com/haya14busa/reviewdog) :dog:</sub>
Metrics/MethodLength: Method has too many lines. [13/10]

**Comment by kyohei0423** [comment]
コメント

**Pull request closed without merging**